website/docs/reference/dbt-jinja-functions/execute.md

```diff
@@ -17,17 +17,17 @@ Any Jinja that relies on a result being returned from the database will error du
 <File name='models/order_payment_methods.sql'>
 
 ```sql
-{% set payment_method_query %}
-select distinct
-payment_method
-from {{ ref('raw_payments') }}
-order by 1
-{% endset %}
-
-{% set results = run_query(payment_method_query) %}
-
-{# Return the first column #}
-{% set payment_methods = results.columns[0].values() %}
+1   {% set payment_method_query %}
```

Comment:
completely agree @ialdg with you, added line numbers here for ease. thanks for that sugg!